packages/shared/web/src/dashboard-view/hooks/use-leverage-points-grid.tsx

```diff
@@ -0,0 +1,137 @@
+import { useState } from 'react';
+import {
+  useAllVaults,
+  useVaultHoldings,
+  useAllMarkets,
+  useFiat,
+} from '@notional-finance/notionable-hooks';
+import { useHistory } from 'react-router';
+import { DashboardGridProps, DashboardDataProps } from '@notional-finance/mui';
+import { Network, PRODUCTS } from '@notional-finance/util';
+import { formatNumberAsAbbr } from '@notional-finance/helpers';
+import { defineMessage } from 'react-intl';
+import { PointsIcon } from '@notional-finance/icons';
+import { Box } from '@mui/material';
+
+export const useLeveragePointsGrid = (network: Network): DashboardGridProps => {
```

Comment:
It seems like the grid hook and the list hook are pretty similar, lots of repeated code. Is there a way to make this more DRY?